src/php/inc/theme-widgets.php

```diff
@@ -6,14 +6,26 @@
  */
 
 /**
- * Register widget area.
+ * Register widgetized areas.
  */
 function sparkpress_theme_widgets_init() {
+	register_sidebar(
+		array(
+			'name'          => 'Primary Widget Area',
+			'id'            => 'primary-widget-area',
+			'description'   => 'Primary sidebar area on the side of the page.',
+			'before_widget' => '<div>',
+			'after_widget'  => '</div>',
+			'before_title'  => '<h3>',
+			'after_title'   => '</h3>',
+		)
+	);
+
 	register_sidebar(
 		array(
 			'name'          => 'Footer',
 			'id'            => 'footer-area',
-			'description'   => 'Sidebar for a the footer area.',
+			'description'   => 'Sidebar for the footer area.',
```

Comment:
I don't actually know where the description gets shown, but can we call this "Widget for the footer area"?